types/hooks/screens/useAnonymousChatInfoScreenHook.types.ts

```diff
@@ -1,9 +1,23 @@
 import {ChannelList, ChannelListMemberSchema} from '../../database/schema/ChannelList.types';
+import {Member} from '../../database/schema/ChatListDetail.types';
 
 interface UseAnonymousChatInfoScreenHook {
   channelInfo: ChannelList;
   goBack: () => void;
-  onContactPressed: (item: ChannelListMemberSchema) => void;
+  onContactPressed: (item: ChannelListMemberSchema, from?: string) => void;
+  isAnonymous: boolean;
+  selectedUser: ChannelListMemberSchema;
+  showPopupBlock: boolean;
+  handleClosePopup: () => void;
+  isAnonymousUser: (item: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `isAnonymousUser` is declared to return void, but based on its name, it seems like it should return a boolean value indicating whether the user is anonymous or not. Please check and correct the return type if necessary.
```diff
-   isAnonymousUser: (item: any) => void;
+   isAnonymousUser: (item: any) => boolean;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->